.circleci/config.yml

```
@@ -353,6 +354,20 @@ jobs:
       - run: cd solidity/ && curl "https://binaries.soliditylang.org/bin/soljson-nightly.js" --location --output soljson.js
       - run: cd solidity/ && test/externalTests/solc-js/solc-js.sh "$(realpath soljson.js)" "$(scripts/get_version.sh)" "$(realpath ../solc-js/)"
 
+  download-release-solc-test:
+    docker:
+      - image: cimg/node:current
+    steps:
+      - show-npm-version
+      - provision-and-package-solcjs
+      - run:
+          name: Check solc version with the released version
+          command: |
+            cd solc-js
+            npm run updateBinary
+            npm run build
+            scripts/check-binary.sh
```

"check" is not very specific. Can we give it a better name?
```suggestion
          name: Verify that `npm run updateBinary` downloads the latest release
          command: |
            cd solc-js
            npm run updateBinary
            npm run build
            scripts/is-binary-up-to-date.sh
```